.azuredevops/pull_request_templates/branches/production.md

```diff
@@ -0,0 +1,9 @@
+## Before deploying to production, please ensure:
+- [ ] Release notes have been sent to dncpartners two days ago and accurately reflect the contents & impact of this rollout
+- [ ] The staging build is green and has been stable for at least two days
+- [ ] The commit message contains '[ROLLOUT]', '[HOTFIX]', or '[ROLLBACK]' as appropriate
+- [ ] The build master has approved the rollout
```

Comment:
Is there a definition for what this means?

Reply:
Yeah, it refers to whatever we want to call Mark's role. When I wrote this I was thinking I shouldn't mention him by name like I did in the internal repos, but actually I think that might b fine? Should I just tag him?

Reply:
Ahh, I actually hadn't associated this with Mark's role. If anything I would call him the deployment master 😁 

Perhaps it's enough to say that it has leadership approval?

Reply:
hahaha deployment master sounds fantastic. I'll go with leadership approval though.